src/nsrt_learning/strips_learning/clustering_learner.py

```diff
@@ -313,6 +313,8 @@ def _postprocessing_learn_side_predicates(
         pnads = list(path[-1])
         # Because the PNADs have been modified, recompute the datastores.
         self._recompute_datastores_from_segments(pnads)
+        # Filter out PNADs that have an empty datastore.
+        pnads = [pnad for pnad in pnads if pnad.datastore]
```

Comment:
fixes some test that was breaking due to the change in score function... i think it's correct anyway